packages/mobile/src/home/reducers.ts

```diff
@@ -1,34 +1,84 @@
-import dotProp from 'dot-prop-immutable'
-import { Actions } from 'src/home/actions'
+import { RehydrateAction } from 'redux-persist'
+import { Actions, ActionTypes } from 'src/home/actions'
+import { getRehydratePayload, REHYDRATE } from 'src/redux/persist-helper'
+
+export interface NotificationTexts {
+  body: string
+  cta: string
+  dismiss: string
+}
+
+export interface Notification {
+  ctaUri: string
+  darkMode: boolean
+  en: NotificationTexts
+  es: NotificationTexts
+  pt: NotificationTexts
```

Comment:
nit: grouping these with the rest makes typing slightly more difficult and error prone as we add more languages.
```suggestion
  content: { [lang: string] : NotificationTexts | undefined }
```

Reply:
That sounds a lot better, I was looking for some way of doing that 😄